packages/tasks/src/model-libraries.ts

```diff
@@ -150,6 +150,18 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path:"adapter_config.json"`,
 	},
+	deepforest_tree: {
+		prettyLabel: "DeepForest Tree",
+		repoName: "deepforest-Tree",
```

Comment:
```suggestion
		repoName: "deepforest-Tree",
		docsUrl: "https://deepforest.readthedocs.io/en/latest/",
```

It's worth adding a link to your official docs as well :)